src/pages/ServerDisplay/ServerDisplay.tsx

```diff
@@ -51,7 +51,15 @@ export default function ServerDisplay() {
     const getServerUsers = async () => {
         try {
             const response = await new ServerService().getServerUsers(urlSearchParams.serverId!);
-            setUsers(response.result);
+            console.log(usersState.data);
+            let usersToAdd: User[] = [];
+            response.result.forEach((elt: User) => {
```

Comment:
le contenu de `response.result` peut avoir des doublons ? Je ne comprends pas trop le besoin de check si l'item sur lequel tu itères est déjà présent dans ton nouveau tableau ou pas sinon

Reply:
Ah Merde ! ^^
C'est pas pour là que j'avais pondu ça, c'est pour ajouter les auteurs des messages dans le state si ils n'y sont pas déjà.
Et donc oui, dans une liste de messages reçus, il peut y avoir plusieurs fois le même auteur, c'est même assez courant.
Bon, du coup je corrige ça, mais ça va pas être facile :(

Reply:
Heu, done.
J'ai enlevé la recherche de doublons.